SHMEM/isx.c

```diff
@@ -101,7 +102,7 @@ int main(const int argc,  char ** argv)
 // to set all necessary runtime values and options
 static char * parse_params(const int argc, char ** argv)
 {
-  if(argc != 3)
+  if(argc < 3 || argc > 4)
```

Comment:
Could you please update the usage message below to mention the number of iterations argument?